lib/geordi/commands/cucumber.rb

```diff
@@ -2,9 +2,8 @@
 long_desc <<-LONGDESC
 Example: `geordi cucumber features/authentication_feature:3`
 
-Runs Cucumber with `bundle exec`, using parallel tests, with a VNC session
-holding Selenium test browsers, support for using a dedicated testing browser
-and beta support for re-running failed scenarios.
+Runs Cucumber with `bundle exec`, using parallel tests, with a headless Chrome session, 
+with beta support for re-running failed scenarios.
```

Comment:
Please do not mention Headless Chrome, as it is a project configuration thing. Geordi fortunately does not have to know about it.

```suggestion
Runs Cucumber with `bundle exec`, using parallel tests and with support for re-running
failed scenarios.
```